doc/api/modules.md

```diff
@@ -373,20 +365,17 @@ LOAD_AS_FILE(X)
       1. MAYBE_DETECT_AND_LOAD(X.js)
     c. If the SCOPE/package.json contains "type" field,
       1. If the "type" field is "module", load X.js as an ECMAScript module. STOP.
-      2. If the "type" field is "commonjs", load X.js as an CommonJS module. STOP.
+      2. If the "type" field is "commonjs", load X.js as a CommonJS module. STOP.
     d. MAYBE_DETECT_AND_LOAD(X.js)
 3. If X.json is a file, load X.json to a JavaScript Object. STOP
 4. If X.node is a file, load X.node as binary addon. STOP
 
-LOAD_INDEX(X)
-1. If X/index.js is a file
-    a. Find the closest package scope SCOPE to X.
-    b. If no scope was found, load X/index.js as a CommonJS module. STOP.
-    c. If the SCOPE/package.json contains "type" field,
-      1. If the "type" field is "module", load X/index.js as an ECMAScript module. STOP.
-      2. Else, load X/index.js as an CommonJS module. STOP.
-2. If X/index.json is a file, parse X/index.json to a JavaScript object. STOP
-3. If X/index.node is a file, load X/index.node as binary addon. STOP
+MAYBE_DETECT_AND_LOAD(X)
```

Comment:
Question: here and in other parts of this pseudo-code there are various anchors, those are rendered as plain text and not actual links, is this intentional?
![Screenshot 2025-02-14 at 23 38 38](https://github.com/user-attachments/assets/b5dc1153-04c2-4bd8-8b3a-753485b48c55)